### DISCUSSION TITLE
AppWrite Plugin

### DISCUSSION THREAD

**Discussion by kunduarnab** [Budibase/budibase#8226] [Ideas]
AppWrite datasource integration or plugin 

Appwrite - https://appwrite.io/

**Comment by melohagan** [comment]
Hey @kunduarnab 

I've added datasources for some of the core Appwrite domains. You can see them here: https://github.com/Budibase/plugins/

**Reply by kunduarnab** [reply]
Okey.. Thank you...let me check...🤘

**Reply by kunduarnab** [reply]
Working perfectly @melohagan

**Reply by melohagan** [reply]
Awesome!
